README.md

```diff
@@ -10,7 +10,7 @@ It was build to serve the needs of the [Cartographer](https://github.com/googlec
 The project consist of a root crate that can build and read octrees on disk and viewer binaries to visualize the data. For Mac OS X, we assume below that you've installed [Homebrew](https://brew.sh).
 
 - Install Rust: `curl https://sh.rustup.rs -sSf | sh`. See <https://rustup.rs> for details.
-- Install protobuf > 3.0. See [ci/install_proto3.sh](https://github.com/googlecartographer/point_cloud_viewer/blob/master/ci/install_proto3.sh) for Linux or run `brew install protobuf` on Mac OS X.
+- Install `protoc` > 3.0 in `target/protobuf/bin`. The [`protobuf` Makefile target](https://github.com/googlecartographer/point_cloud_viewer/blob/master/Makefile) will compile `protoc` from source and place it there. Alternatively, you can symlink to a `protoc` installed on your system, e.g. from `brew install protobuf` on Mac OS X.
```

Comment:
make the separate steps for Linux vs. macOS clearer. Something along these lines:
- Linux: Install protoc by running the makefile
- macOS: brew install protobuf, symlink from ... to ...

Reply:
Do you agree with the current formulation? The script works for both Mac and Linux.